lib/scrabble_scoring.rb

```diff
@@ -0,0 +1,76 @@
+module Scrabble
+  class Scoring
+    LETTERS = {
```

Comment:
Dan, I did test on 7 letter word for Scoring#score. (you said "no")   I guess it didn't seem obvious...